src/Backend/Modules/Mailmotor/Config.php

```diff
@@ -156,7 +174,12 @@ private function checkForGroups()
         // check if there are external groups present in CampaignMonitor
         if ($this->checkForExternalGroups()) {
             // external groups were found, so redirect to the import_groups action
-            \SpoonHTTP::redirect(BackendModel::createURLForAction('ImportGroups', 'Mailmotor'));
+            throw new RedirectException(
```

Comment:
A config file should be able to use `$this->redirect` IMO. I'd create a redirect method in the BaseConfig file, that calls this code. It's pretty common to redirect from a config file, so I would hide this Exception hack for the developers and just add a clean $this->redirect.